Panic during an object put with a meta enabled container

```
debug	get/get.go:114	received api response locally, return directly	{"component": "Object.Get service", "address": "4LUGgdwQzPy9mCeAnZCxBHXBvo1GXqPddjvVRDh8Rdzz/DmvjL7m2vC4pd3m582PvxGsSonoxt3VwmbPQrBUpPUY4", "raw": false, "local": false, "with session": false, "with bearer": false, "request": "HEAD", "error": "read object header from NeoFS: /dns4/localhost/tcp/50269: status: code = 3072 message = container not found"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x104f36100]

goroutine 435 [running]:
main.(*neofsNetwork).Head(_, {_, _}, {0x31, 0x8f, 0xee, 0x4f, 0x20, 0x1, 0x6e, ...}, ...)
	github.com/nspcc-dev/neofs-node/cmd/neofs-node/meta.go:86 +0x170
github.com/nspcc-dev/neofs-node/pkg/services/meta.(*containerStorage).putRawIndexes(0x1400049e780, {0x1054ac088, 0x14000135ec0}, 0x1400055b300, {0x140001b6c00, 0x1, 0x1?}, {0x1054ab998, 0x140002ca1b0}, 0x140001ee2a0)
	github.com/nspcc-dev/neofs-node/pkg/services/meta/containers.go:155 +0x3b4
github.com/nspcc-dev/neofs-node/pkg/services/meta.(*containerStorage).putObjects.func1()
	github.com/nspcc-dev/neofs-node/pkg/services/meta/containers.go:73 +0x7c
created by github.com/nspcc-dev/neofs-node/pkg/services/meta.(*containerStorage).putObjects in goroutine 434
	github.com/nspcc-dev/neofs-node/pkg/services/meta/containers.go:71 +0x160
```

https://rest.fs.neo.org/HXSaMJXk2g8C14ht8HSi7BBaiYZ1HeWh2xnWPGQCg4H6/996-1743718238/index.html#suites/0e3180b9561587482b2f1488d7674deb/750c699e77d2004c/

--- a/cmd/neofs-node/meta.go
+++ b/cmd/neofs-node/meta.go
@@ -82,8 +82,11 @@ func (c *neofsNetwork) Head(ctx context.Context, cID cid.ID, oID oid.ID) (object
 	hPrm.WithAddress(oid.NewAddress(cID, oID))
 
 	err := c.header.Head(ctx, hPrm)
+	if err != nil {
+		return object.Object{}, err
+	}
 
-	return *hw.h, err
+	return *hw.h, nil
 }
 
 func (c *neofsNetwork) IsMineWithMeta(id cid.ID) (bool, error) {

